Updated Right Triangle

GrafGeometr opened the pull request:
I replaced your code in GeoGenIntegration from this
```
"RightTriangle" -> {
    check(names.size == 3) { "Expected 3 points" }
    triangles["triangle(${names.joinToString()})"] = "t"
    """
        ${names[0]} = #(-60-20i)
        ${names[1]} = #(60-20i)
        ${names[2]} = project(#(-60+60i), perpendicular(${names[0]}, ${names[0]}, ${names[1]}))
        [orange] [fill] t = triangle(${names.joinToString()})
    """.trimIndent()
}
```
to this
```
"RightTriangle" -> {
    check(names.size == 3) { "Expected 3 points" }
    triangles["triangle(${names.joinToString()})"] = "t"
    """
        ${names[1]} = #(-60-20i)
        ${names[2]} = #(60-20i)
        ${names[0]} = cproject(#(-20), midpoint(${names[1]}, ${names[2]}), ${names[1]})
        [orange] [fill] t = triangle(${names.joinToString()})
    """.trimIndent()
}
```
I really hope these changes will improve the quality of your application

GrafGeometr committed:
Update Right Triangle

ilyasm0919 commented:
Please replace circle through the midpoint with the more convinient `diameter_circle`

GrafGeometr committed:
Updated RightTriangle

Circle through midpoint replaced to diameter_circle and added right angle display.

GrafGeometr commented:
Ok, I replaced circle through the midpoint with `diameter_circle`. I've also added a right angle display.
